project-minimization-plugin/src/main/kotlin/org/plan/research/minimization/plugin/settings/AppSettingsConfigurable.kt

```diff
@@ -42,21 +39,18 @@ class AppSettingsConfigurable(private val project: Project) : Configurable {
     }
 
     override fun apply() {
-        val newState = MinimizationPluginState().apply {
-            compilationStrategy = mySettingsComponent?.compilationStrategy ?: CompilationStrategy.GRADLE_IDEA
-            gradleTask = mySettingsComponent?.gradleTask ?: "build"
-            gradleOptions = mySettingsComponent?.gradleOptions ?: emptyList()
-            temporaryProjectLocation = mySettingsComponent?.temporaryProjectLocation ?: "minimization-project-snapshots"
-            snapshotStrategy = mySettingsComponent?.snapshotStrategy ?: SnapshotStrategy.PROJECT_CLONING
-            exceptionComparingStrategy = mySettingsComponent?.exceptionComparingStrategy ?: ExceptionComparingStrategy.SIMPLE
-            stages = mySettingsComponent?.stages ?: listOf(
-                FunctionLevelStage(),
-                FileLevelStage(),
-            )
-            minimizationTransformations = mySettingsComponent?.transformations ?: listOf(
-                TransformationDescriptors.PATH_RELATIVIZATION,
-            )
-        }
+        val newState = mySettingsComponent?.let {
+            MinimizationPluginState().apply {
+                compilationStrategy = mySettingsComponent!!.compilationStrategy
+                gradleTask = mySettingsComponent!!.gradleTask
+                gradleOptions = mySettingsComponent!!.gradleOptions
+                temporaryProjectLocation = mySettingsComponent!!.temporaryProjectLocation
+                snapshotStrategy = mySettingsComponent!!.snapshotStrategy
+                exceptionComparingStrategy = mySettingsComponent!!.exceptionComparingStrategy
+                stages = mySettingsComponent!!.stages
+                minimizationTransformations = mySettingsComponent!!.transformations
```

Comment:
I suggest using this one. It is more kotlin stylish :)

```suggestion
                compilationStrategy = it.compilationStrategy
                gradleTask = it.gradleTask
                gradleOptions = it.gradleOptions
                temporaryProjectLocation = it.temporaryProjectLocation
                snapshotStrategy = it.snapshotStrategy
                exceptionComparingStrategy = it.exceptionComparingStrategy
                stages = it.stages
                minimizationTransformations = it.transformations
```